python/ray/local_mode_manager.py

```diff
@@ -20,7 +21,18 @@ class LocalModeObjectID(ObjectID):
             it equates to the object not existing in the object store. This is
             necessary because None is a valid object value.
     """
-    pass
+
+    def __copy__(self):
+        new = LocalModeObjectID(self.binary())
+        if hasattr(self, "value"):
+            new.value = self.value
+        return new
+
+    def __deepcopy__(self, memo={}):
```

Comment:
can you take out the dict? https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments